js_modules/dagster-ui/packages/ui-components/src/components/ButtonLink.tsx

```diff
@@ -51,6 +52,9 @@ const textDecoration = (underline: Underline) => {
       return css`
         &:hover {
           text-decoration: underline;
+          & > ${Box} {
+            text-decoration: underline;
+          }
```

Comment:
This is a global change but I noticed this hover effect isn't working in many places because a `<Box>`(for an icon and text side by side) doesn't inherit the decoration for reasons unknown